postgres-size-report

```diff
@@ -36,6 +36,12 @@ echo ""
 echo $TABLEQUERY | sudo -u postgres psql -d foreman
 
 
+echo ""
+echo "************** Pulpcore Tablesizes *************"
+echo ""
+echo $TABLEQUERY | sudo -u postgres psql -d pulpcore
```

Comment:
Hey @math3matical , thanks for the PR!

I did some further thinking about how we can support Satellite versions that don't have this database. While `psql: error: FATAL:  database "pulpcore" does not exist` isn't the end of the world and the script would continue executing, we can prevent that ugly and potentially confusing output using a nice conditional:

```suggestion
if PULPCORE_TABLESIZES=$(echo $TABLEQUERY | sudo -u postgres psql -d pulpcore 2>/dev/null)
then
  echo ""
  echo "************** Pulpcore Tablesizes *************"
  echo ""
  echo $PULPCORE_TABLESIZES
fi
```